backend/lombok.config

```diff
@@ -1,2 +1,3 @@
 config.stopBubbling = true
 lombok.copyableAnnotations += org.springframework.beans.factory.annotation.Qualifier
+lombok.addLombokGeneratedAnnotation = true
```

Comment:
This effectively makes sonar not ask for test coverage on lombok annotation, such as @Value for example.